src/fundus/publishers/de/rbb24.py

```diff
@@ -0,0 +1,95 @@
+import datetime
+from typing import List, Optional
+
+from lxml.cssselect import CSSSelector
+from lxml.etree import XPath
+
+from fundus.parser import ArticleBody, BaseParser, ParserProxy, attribute
+from fundus.parser.utility import (
+    extract_article_body_with_selector,
+    generic_author_parsing,
+    generic_date_parsing,
+)
+
+
+class RBB24Parser(ParserProxy):
+    class V1(BaseParser):
+        _paragraph_selector = XPath("//div[contains(concat(' ', @class , ' '), ' textblock ')]")
+        _subheadline_selector = XPath("//h4[contains(concat(' ', @class , ' '), ' texttitle ')]")
+        _author_selector = CSSSelector("span.authorname")
+        _date_selector = CSSSelector("div.lineinfo")
+
+        @attribute
+        def body(self) -> ArticleBody:
+            article_body = extract_article_body_with_selector(
+                self.precomputed.doc,
+                summary_selector=XPath("//div[contains(concat(' ', @class , ' '), ' shorttext ')]/p"),
```

Comment:
For more clarity, could you move this selector up to where you define the other variables as well?